### ISSUE TITLE
Logarithmic differentiation in MTK

### ISSUE THREAD

**Issue by finmod** [SciML/ModelingToolkit.jl#1344]
Can MTK handle in both symbolic and ODE solver form logarithmic differentiation?  https://en.wikipedia.org/wiki/Logarithmic_differentiation#:~:text=In%20calculus%2C%20logarithmic%20differentiation%20or,rather%20than%20the%20function%20itself. 

A typical solution to linearize a nonlinear ODE system:
Y = K/v and Differential (K) = I - bK ; a is a parameter then
Differential (ln Y) = Differential (Y)/Y = I/vY - b

**Comment by ChrisRackauckas** [comment]
Yeah that's just a pass that needs to be written. The underlying symbolic pieces should handle it fine.
